cdc/capture/capture.go

```diff
@@ -60,6 +61,9 @@ type Capture struct {
 	etcdClient *etcd.CDCEtcdClient
 	grpcPool   kv.GrpcPool
 
+	tableActorSystem *actor.System
+	tableActorRouter *actor.Router
```

Comment:
Could you wrap them in a struct? like https://github.com/pingcap/ticdc/pull/3406

Reply:
ok